packages/@lwc/synthetic-shadow/src/faux-shadow/event-target.ts

```diff
@@ -18,4 +17,73 @@ const eventCurrentTargetGetter: (this: Event) => Element | null = getOwnProperty
     'currentTarget'
 )!.get!;
 
+// These methods are usually from EventTarget.prototype, but that's not available in IE11, the best best thing
+// is Node.prototype, which is an EventTarget as well.
+const {
+    dispatchEvent,
+    addEventListener: superAddEventListener,
+    removeEventListener: superRemoveEventListener,
+} = Node.prototype;
+
+function addEventListenerPatched(
+    this: Element,
+    type: string,
+    listener: EventListener,
+    options?: boolean | AddEventListenerOptions
+) {
+    if (hasSyntheticShadow(this)) {
+        addCustomElementEventListener(this, type, listener, options);
+    } else {
+        superAddEventListener.call(this, type, listener, options);
+    }
+}
+
+function removeEventListenerPatched(
+    this: Element,
+    type: string,
+    listener: EventListener,
+    options?: boolean | AddEventListenerOptions
+) {
+    if (hasSyntheticShadow(this)) {
+        removeCustomElementEventListener(this, type, listener, options);
+    } else {
+        superRemoveEventListener.call(this, type, listener, options);
+    }
+}
+
+// IE11 doesn't have EventTarget, so we have to patch it conditionally:
+
+if (typeof EventTarget !== 'undefined') {
```

Comment:
`if (!isUndefined(typeof EventTarget))`

Reply:
It is not the same... typeof always return string values, while isUndefined() tests for === with `undefined`